src/main/java/ca/fxco/configurablepistons/helpers/Utils.java

```diff
@@ -52,4 +56,11 @@ public static boolean hasNeighborSignalExceptFromFacing(Level level, BlockPos po
 
         return false;
     }
+
+    public static boolean setBlockWithEntity(Level level, BlockPos blockPos,
```

Comment:
For a general utility method maybe the set block flags should be passed as a parameter?